google/cloud/bigquery/v2/minimal/internal/dataset_response_test.cc

```diff
@@ -166,6 +167,293 @@ TEST(ListDatasetsResponseTest, InvalidListFormatDataset) {
                        HasSubstr("Not a valid Json ListFormatDataset object")));
 }
 
+TEST(GetDatasetResponseTest, DebugString) {
+  BigQueryHttpResponse http_response;
+  http_response.http_status_code = HttpStatusCode::kOk;
+  http_response.http_headers.insert({{"header1", "value1"}});
+  http_response.payload =
+      R"({"kind": "d-kind",
+          "etag": "d-tag",
+          "id": "d-id",
+          "self_link": "d-selfLink",
+          "friendly_name": "d-friendly-name",
+          "dataset_reference": {"project_id": "p-id", "dataset_id": "d-id"}
+    })";
+  auto response = GetDatasetResponse::BuildFromHttpResponse(http_response);
+  ASSERT_STATUS_OK(response);
+
+  EXPECT_EQ(response->DebugString("GetDatasetResponse", TracingOptions{}),
+            R"(GetDatasetResponse {)"
+            R"( dataset {)"
+            R"( kind: "d-kind")"
+            R"( etag: "d-tag")"
+            R"( id: "d-id")"
+            R"( self_link: "d-selfLink")"
+            R"( friendly_name: "d-friendly-name")"
+            R"( description: "")"
+            R"( type: "")"
+            R"( location: "")"
+            R"( default_collation: "")"
+            R"( published: false)"
+            R"( is_case_insensitive: false)"
+            R"( default_table_expiration: 0 )"
+            R"(default_partition_expiration: 0)"
```

Comment:
It would be nice to continue the convention of placing the separator at the beginning of this line rather than the end of the previous one (here and in a handful of following cases).